lib/semantics/resolve-names.cc

```diff
@@ -829,7 +829,7 @@ class DeclarationVisitor : public ArraySpecVisitor,
   Attrs HandleSaveName(const SourceName &, Attrs);
   void AddSaveName(std::set<SourceName> &, const SourceName &);
   void SetSaveAttr(Symbol &);
-  bool HandleUnrestrictedSpecificIntrinsicFunction(const parser::Name &);
+  Symbol *HandleUnrestrictedSpecificIntrinsicFunction(const parser::Name &);
```

Comment:
I don't understand changing the return type. It looks like `NameIsKnownOrIntrinsic` and `HandleUnrestrictedSpecificIntrinsicFunction` are still only used in a boolean context.

Reply:
I'll change it back then.